packages/cli/cloud/src/deploy-project/action.ts

```diff
@@ -281,6 +300,26 @@ export default async (ctx: CLIContext, opts: CmdOptions) => {
 
   project.targetEnvironment = await getTargetEnvironment(ctx, opts, project, environments);
 
+  if (!opts.force) {
+    const shouldDisplayWarning = await hasPendingOrLiveDeployment(
```

Comment:
does it need to be await/async?

Reply:
Good catch, it doesn't seem needed since we're not performing async operations in the function. Correcting.